src/components/action/validation/validationActionComponents/ValidationSettings.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+
+import CheckBox from '../../checkBox/CheckBox';
+import './ValidationSettings.css';
+import ValidationProfile from './ValidationProfile';
+import FixMetadata from './FixMetadata';
+
+class ValidationSettings extends Component {
+    fileInput = null;
+
+    render() {
+        const titleInformation =
```

Comment:
Why here and not in Checkbox directly like you did for e.g. "Fix metadata"?
I usually tend to create a variable when it depends on smth, otherwise if you don't want to use it hardcoded consider to move into constant outside the class.